libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/voice/AndroidSpeechPlayer.java

```diff
@@ -21,6 +23,7 @@
 
```

Comment:
I guess `public` could be removed ☝️
The same thing applies to `MapboxSpeechPlayer` and `InstructionPlayer`.

Reply:
> I guess `public` could be removed ☝️
> The same thing applies to `MapboxSpeechPlayer` and `InstructionPlayer`.

`public` could be removed from `InstructionPlayer`.